proposals/interface-types/working-notes/Instructions.md

```diff
@@ -0,0 +1,369 @@
+# Interface instructions
+
+This proposal defines new instructions that operate over interface value types. These instructions have the same form as core instructions; they push and pop values on an execution stack and support single pass validation.
+
+The new interface instructions are summarized below.
+
+```
+#interface-instr ::=
+  | bool.lift_iMM
+  | bool.lower_iMM
+
+  | sNN.lift_iMM
+  | sNN.lower_iMM
+
+  | uNN.lift_iMM
+  | uNN.lower_iMM
+
+  | string.lift_memory ..
+  | string.lower_memory ..
+
+  | variant.lift ..
+  | variant.lower_tag ..
+  | variant.lower ..
+
+  | record.lift ..
+  | record.lower ..
+
+  | array.lift_memory ..
+  | array.lower_memory ..
+
+  | callback.lift ..
+  | callback.lower ..
+
+  | defer ..
+
+where:
+  NN is one of [8, 16, 32, 64]
+  MM is one of [32, 64]
```

Comment:
We had agreed to distinguish signed and unsigned variants (sic) of integral types

Reply:
The values are signed/unsigned distinguished, see `sNN/uNN`.